<?xml version="1.0"?>
<launch>	
	<node name="vs_evaluate" pkg="gcop_ctrl" type="vs_evaluate" output="screen">
    <remap from="image" to="/camera/image_raw" />
    <remap from="camera_info" to="/camera/camera_info" />
    <!--<remap from="depth" to="/mesh_localize/depth" />-->
    <param name="im_goal_filename" type="string" value="$(find gcop_ctrl)/data/frame0006.jpg" />
  </node>
</launch>
